css/style.css

```diff
@@ -0,0 +1,315 @@
+@import url('https://fonts.googleapis.com/css2?family=Outfit:wght@300&display=swap');
+
+:root {
+    --main-light: #94618E; 
+    --main-dark: #49274A;
+    --secondary-light: #F8EEE7;
+    --secondary-dark: #F4DECB;
+    --grey-dark: #848993;
+    --grey-darker: #4D5057;
+}
+* {
+    margin: 0;
+    font-family:'Outfit', sans-serif;
+}
+header {
+    background-color: var(--main-light);
+    display: grid;
+    grid-template-columns: 1fr 0.3fr 1fr 1fr;
+    grid-template-areas:
+    "logo . . toggle"
+    "logo . text text"
+    "logo . text text"
+    ". . . ."
+    "nav nav nav nav";
+    color: var(--secondary-light);
+ 
+}
+.sticky {
+    position:fixed;
+}
+.logo {
+    grid-area: logo;
+    display: flex;
+    align-items: center;
+    margin: 1rem 2rem;
+    z-index: 2;
+}
+.logo img {
+    height:6rem;
+}
+.menu-toggle {
+    grid-area: toggle;
+    text-align: right;
+    margin: 1rem 2rem 0 0;
+    z-index: 2;
+}
+.menu-toggle svg {
+    height: 2rem;
+    fill: var(--secondary-light);
+    stroke: var(--secondary-light);
+}
+header h1 {
+    grid-area: text;
+    font-size: 1.5rem;
+    text-align: right;
+    display:flex;
+    justify-content: flex-end;
+    align-items: flex-end;
+    margin: 0 2rem 1rem 0;
+    z-index: 2;
+}
+nav {
+    display: flex;
+    grid-area: nav;
+    position: fixed;
+    height: 100vh;
+    padding-top:8rem;
+    width:100%;    
+    background-color: var(--main-light);
+    clip-path: circle(0px at 95% -5%);
+    -webkit-clip-path: circle(0px at 95% -5%);
+    transition: all 0.5s ease-out;
+}
+nav.open {
+    clip-path: circle(150vh at 95% -5%);
+    -webkit-clip-path: circle(150vh at 95% -5%);
+}
+nav ul {
+    list-style: none;
+    width: 100%;
+    padding:0;
+}
+nav li a:hover {
+    background-color: var(--main-dark);
+}
+.navlink {
+    padding: 0.5rem 0;
+    text-align: center;
+}
+.navlink a{
+    color: var(--secondary-light);
+    text-decoration: none; 
+    width: 100%;
+    display: block;
+    padding: 1rem 0;
+}
+h2 {
+    font-size: 4rem;
+    font-weight: 600;
+    padding-bottom: 2rem;
+}
+main {
+    display: grid;
+    grid-auto-flow: dense;
+    grid-gap:0.5rem;
+    grid-template-columns: repeat(12, 1fr);
```

Comment:
Love the 12 columns layout, to give you extra flexibility in positing content, well done!